packages/ui/src/styles/tokens.css

```
@@ -26,26 +26,29 @@
   --color-gray-700: #7f828c;
   --color-gray-800: #5a5e6a;
   --color-gray-900: #3b3f4a;
+
+  /* Etc Colors */
+  --color-navy: #14181f;
 }
 
 @font-face {
-  font-family: "Pretendard";
-  src: url("./fonts/Pretendard-Bold.otf") format("opentype");
+  font-family: 'Pretendard';
```

Stylelint (17.3.0)

[error] Unexpected quotes around "Pretendard" (font-family-name-quotes)

(font-family-name-quotes)